.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+# Keep GitHub Actions up to date with GitHub's Dependabot...
+# https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot
+# https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    groups:
+      github-actions:
+        patterns:
+          - "*" # Group all Actions updates into a single larger pull request
+    schedule:
+      interval: weekly
```

Comment:
Maybe let's set the interval to monthly. Weekly PRs that update GHA are sometimes cumbersome, and for that reason, we've moved to pre-commit-config updates from weekly to quarterly as well.

Reply:
I see. That makes sense. Thanks @cclauss!